ui/search.js

```diff
@@ -95,13 +95,24 @@ function setInternetSearchProviderEnable(enabled) {
 }
 
 function enable() {
-    Utils.override(Search.SearchResult, 'hide', function () {
+    Utils.override(Search.SearchResult, 'activate', function () {
```

Comment:
Maybe not one for today but I feel like `Utils.override(o, n, f)` should warn if `o[n] === undefined`.